crates/sui-open-rpc/Cargo.toml

```diff
@@ -25,6 +25,7 @@ sui-json-rpc = { path = "../sui-json-rpc" }
 sui-json-rpc-types = { path = "../sui-json-rpc-types" }
 sui-json = { path = "../sui-json" }
 sui-types = { path = "../sui-types" }
+sui-sdk = { path = "../sui-sdk" }
```

Comment:
adding dependencies here is not ideal, i would like eventually take the Keystore logic into its own crate from sui-sdk.